Tests failing with graphql-core pre-release version 3.3.0a3

`make tests` results:

```python
FAILED tests/custom_scalars/test_money.py::test_custom_scalar_subscribe_in_input_variable_values_serialized - TypeError: object async_generator can't be used in 'await' expression
FAILED tests/starwars/test_subscription.py::test_subscription_support - TypeError: object async_generator can't be used in 'await' expression
FAILED tests/starwars/test_subscription.py::test_subscription_support_using_client - TypeError: object async_generator can't be used in 'await' expression
FAILED tests/starwars/test_subscription.py::test_subscription_support_using_client_invalid_field - TypeError: object ExecutionResult can't be used in 'await' expression
```

I'm curious why graphql-core was bumped to 3.3.0 alpha. This is causing a problem for my project because I need the HTTPX transport, but graphql-core 3.3.0a conflicts with apischema. This is the stack trace apischema throws:

```bash
    from apischema import ValidationError, deserialize  # type: ignore
/Volumes/workspace/my-project/.venv/lib/python3.11/site-packages/apischema/__init__.py:71: in <module apischema>
    ???
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
/Volumes/workspace/my-project/.venv/lib/python3.11/site-packages/apischema/graphql/__init__.py:14: in <module apischema.graphql>
    ???
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
/Volumes/workspace/my-project/.venv/lib/python3.11/site-packages/apischema/graphql/relay/__init__.py:15: in <module apischema.graphql.relay>
    ???
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
/Volumes/workspace/my-project/.venv/lib/python3.11/site-packages/apischema/graphql/relay/global_identification.py:21: in <module apischema.graphql.relay.global_identification>
    ???
<frozen importlib._bootstrap>:1178: in _find_and_load
    ???
<frozen importlib._bootstrap>:1149: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
/Volumes/workspace/my-project/.venv/lib/python3.11/site-packages/apischema/graphql/schema.py:485: in <module apischema.graphql.schema>
    ???
/Volumes/workspace/my-project/.venv/lib/python3.11/site-packages/apischema/graphql/schema.py:487: in InputSchemaBuilder
    ???
E   AttributeError: module 'graphql.type.definition' has no attribute 'graphql_input_types'
```
Unfortunately my project management framework `poetry` is a stickler about dependency resolution with no built-in mechanism for overriding, but if I got into my virtualenv shell and manually replace graphql-core 3.3.0a3 with 3.2.3, the error resolves, my unit tests pass, the `gql` client seems to work fine at first glance, I can establish an async session with my HTTPX transport, call connect_async() and authenticate to my service, and apischema is happy again.

--- a/gql/transport/local_schema.py
+++ b/gql/transport/local_schema.py
@@ -59,7 +59,7 @@ class LocalSchemaTransport(AsyncTransport):
         The results are sent as an ExecutionResult object
         """
 
-        subscribe_result = await subscribe(self.schema, document, *args, **kwargs)
+        subscribe_result = subscribe(self.schema, document, *args, **kwargs)
 
         if isinstance(subscribe_result, ExecutionResult):
             yield subscribe_result

--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,7 @@ import os
 from setuptools import setup, find_packages
 
 install_requires = [
-    "graphql-core>=3.3.0a2,<3.4",
+    "graphql-core>=3.3.0a3,<3.4",
     "yarl>=1.6,<2.0",
     "backoff>=1.11.1,<3.0",
 ]

--- a/tests/starwars/test_subscription.py
+++ b/tests/starwars/test_subscription.py
@@ -30,7 +30,7 @@ async def test_subscription_support():
     params = {"ep": "JEDI"}
     expected = [{**review, "episode": "JEDI"} for review in reviews[6]]
 
-    ai = await subscribe(StarWarsSchema, subs, variable_values=params)
+    ai = subscribe(StarWarsSchema, subs, variable_values=params)
 
     result = [result.data["reviewAdded"] async for result in ai]
 

